src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int side;
+    private int base;
+    private String color = getColor();
+
+    public RightTriangle(int side, int base, String color) {
+        this.side = side;
+        this.base = base;
+        this.color = color;
+    }
+
+    @Override
+    public double areaCalculator() {
+        return (double) (side * base) / 2;
```

Comment:
The method name `areaCalculator` sounds like a verb, which is not recommended for method names. Consider renaming it to something like `calculateArea` to better represent its purpose as a method.